app/models/user.rb

```diff
@@ -5,5 +5,5 @@ class User < ApplicationRecord
   has_many :bookings, dependent: :destroy
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
-  validates :first_name, :last_name, :email, :password, presence: true
```

Comment:
@emanuelmoniz  shouldn't we put the uniqueness of email?

Reply:
devise does that...i dont know where but its already in place

Reply:
I will merge it then